packages/jest-editor-support/src/Process.js

```diff
@@ -27,16 +28,21 @@ export const createProcess = (
   // any other bits into the args
   const runtimeExecutable = workspace.pathToJest;
   const parameters = runtimeExecutable.split(' ');
-  const command = parameters[0];
   const initialArgs = parameters.slice(1);
-  const runtimeArgs = [].concat(initialArgs, args);
+  let command = parameters[0];
+  let runtimeArgs = [].concat(initialArgs, args);
 
   // If a path to configuration file was defined, push it to runtimeArgs
   if (workspace.pathToConfig) {
     runtimeArgs.push('--config');
     runtimeArgs.push(workspace.pathToConfig);
   }
 
+  if (workspace.useWsl) {
+    runtimeArgs = [command, ...runtimeArgs.map(convertWslPath)];
```

Comment:
Just for consistency I would also convert `command` such that we also support absolute commands.